tests/providers/amazon/aws/waiters/test_bedrock.py

```diff
@@ -0,0 +1,71 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+from unittest import mock
+
+import boto3
+import botocore
+import pytest
+
+from airflow.providers.amazon.aws.hooks.bedrock import BedrockHook
+from airflow.providers.amazon.aws.sensors.bedrock import BedrockCustomizeModelCompletedSensor
+
+
+class TestBedrockCustomWaiters:
+    def test_service_waiters(self):
+        assert "model_customization_job_complete" in BedrockHook().list_waiters()
+
+
+class TestBedrockCustomWaitersBase:
```

Comment:
There are no test methods in this test class? Just an unused fixture? Or am I missing something?

Reply:
I jumped the gun on that a little.   I'm already implementing more waiters and they'll be sharing that same fixture (which is used below [on L45](https://github.com/apache/airflow/pull/38693/files#diff-9d844f38bae728c36b62862b4197ffc0d54b238cf85d7841b40d7aa8571acf6cR45) )to mock the boto client.

Reply:
Ahh I see, so this base class will be used by future waiter test classes?

Reply:
I think maybe the autouse is throwing you off.  It creates `self.client` which is then inherited/used/modified in the mock_get_job in TestModelCustomizationJobCompleteWaiter.

In addition, it will also be used by TestProvisionThroughputJobCompleteWaiter which I will have a PR up for either later today or Monday.   I know we generally don't create the helper until it is used a second time, but I got ahead of myself on this one, but I assure you it will get reused :P

Reply:
Yeah, the autouse made sense after your previous message. My last one was just referencing that you have a base class only used once, but as suspected and you've confirmed, more uses are coming. So it's all good :)